code/particle/particle.cpp

```diff
@@ -439,7 +440,20 @@ namespace particle
 
 			Assert( cur_frame < part->nframes );
 
-			batching_add_volume_bitmap(framenum + cur_frame, &pos, part->particle_index % 8, part->radius, alpha);
+			if (part->length) {
```

Comment:
Please make it specific that 0 is the magic value here.
```suggestion
			if (part->length == 0.0f) {
```